### PR TITLE
fixing verb

### PR DESCRIPTION
**Pull request by sstevens2**
This PR matches the verbs.

I found this sentence a little hard to read.  Specifically I found optional phrase "and must do so before"  difficult to follow the sentence. After rereading it a couple of times without the phrase it made sense but wondering if we can move that phrase to make it easier to read?  Maybe to another sentence following?  

My best effort at rearranging but could still use some work.
> Before we can apply concepts and create something new, we must attain the ability to remember and distinguish between new concepts.  Remembering and distinguishing is also a task that is often faster to obtain then applying or creating.

### CODE REVIEW

File: episodes/05-objectives.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ or because they are interesting to you from a theoretical perspective.
 
 The desired end point (the objective) of a lesson should be new things that the learner can do.
 Cognitive skills cannot be equally easily acquired:
-we can attain the abilty to remember and distinguishing between new concepts more quickly than,
+we can attain the abilty to remember and distinguish between new concepts more quickly than,
```
Comment (sstevens2) [inline episodes/05-objectives.md:32]:
```suggestion
we can attain the ability to remember and distinguish between new concepts more quickly than,
```

**Comment by tobyhodges** [comment]
How about this?

> Before we can apply concepts and create something new, we must attain the ability to remember and distinguish between new concepts. Remembering and distinguishing are also abilities that are often faster to gain then applying or creating.

**Comment by tobyhodges** [comment]
Furthermore, I wonder if this benefits from a bit more explanation, such as:

> Before we can apply concepts and create something new, we must attain the ability to remember and distinguish between new concepts: these skills are essential components of more creative tasks, and cannot be skipped past. Remembering and distinguishing are also abilities that are often faster to gain then applying or creating.

**Comment by sstevens2** [comment]
Like both changes.  I think the deeper explanation is outlined to be included later in this episode, so either option should work.

**Pull request merged**